.travis.yml

```diff
@@ -38,3 +47,7 @@ script:
 - ./gradlew jacocoFullReport coveralls
 - (cd wherehows-docker && ./build.sh latest)
 - (cd wherehows-docker && docker-compose config)
+- (cd $WHEREHOWS_DIR && yarn install --non-interactive && node_modules/bower/bin/bower install && npm run-script coverage)
```

Comment:
Can probably replace this by a simple "./gradle wherehows-frontend:emberCoverage" command?

Reply:
I think I tried to create a task for this here: https://github.com/linkedin/WhereHows/pull/882/files#diff-d85ecd1fb6f9e688a940f35c6308aeb7R45
But I encountered an issue with this when I updated the `distZip` task, maybe you can help, the build seems to fail because the output from `emberBuild` wasn't available?